src/MoBi.UI/Views/IndividualBuildingBlockView.cs

```diff
@@ -0,0 +1,213 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Linq;
+using System.Linq.Expressions;
+using System.Windows.Forms;
+using DevExpress.XtraEditors;
+using DevExpress.XtraEditors.Controls;
+using DevExpress.XtraEditors.Repository;
+using DevExpress.XtraLayout;
+using DevExpress.XtraLayout.Utils;
+using MoBi.Presentation.DTO;
+using MoBi.Presentation.Formatters;
+using MoBi.Presentation.Presenter;
+using MoBi.Presentation.Views;
+using OSPSuite.Assets;
+using OSPSuite.Core.Domain.UnitSystem;
+using OSPSuite.DataBinding;
+using OSPSuite.DataBinding.DevExpress;
+using OSPSuite.DataBinding.DevExpress.XtraGrid;
+using OSPSuite.Presentation.DTO;
+using OSPSuite.Presentation.Extensions;
+using OSPSuite.UI.Controls;
+using OSPSuite.UI.Extensions;
+using OSPSuite.UI.RepositoryItems;
+using OSPSuite.Utility.Extensions;
+using static MoBi.Assets.AppConstants.Captions;
+
+namespace MoBi.UI.Views
+{
+   public partial class IndividualBuildingBlockView : BaseUserControl, IIndividualBuildingBlockView
```

Comment:
In the end, refactoring this view to get common code pulled up while maintaining a non-generic base class was unsuccessful. I just couldn't justify any more time on it.

Maybe you have a tip that's going to unlock it for me, but I just couldn't find it.